steampy/client.py

```diff
@@ -181,7 +184,11 @@ def accept_trade_offer(self, trade_offer_id: str) -> dict:
     def _fetch_trade_partner_id(self, trade_offer_id: str) -> dict:
```

Comment:
You can change type to def accept_trade_offer(self, trade_offer_id: str) -> str: 
I spotted that bug when i was reviewing your pull request